base/src/main/scala/data/Const.scala

```diff
@@ -0,0 +1,8 @@
+package scalaz
+package data
+
+final case class  Const[A, B](getConst: A){
+  def retag[C]: Const[A, C] = this.asInstanceOf[Const[A, C]]
```

Comment:
We should use `liftCvf` as described in "[When can Liskov be lifted?](http://typelevel.org/blog/2014/03/09/liskov_lifting.html)"